Lib/xml/dom/pulldom.py

```diff
@@ -172,14 +174,17 @@ def buildDocument(self, uri, tagname):
         # Put everything we have seen so far into the document
         for e in self.pending_events:
             if e[0][0] == PROCESSING_INSTRUCTION:
-                _,target,data = e[0]
-                n = self.document.createProcessingInstruction(target, data)
+                _, target, data = e[0]
+                n = self.document.createProcessingInstruction(target,
```

Comment:
This line split is unnecessary.

Reply:
`autopep8` complains if I don't split it.